static/src/stylesheets/module/crosswords/_clues.scss

```diff
@@ -36,12 +38,8 @@
     color: colour(neutral-2);
 }
 
-.crossword__clue--display-group-order {
-
-    padding-left: 0px;
-
-    //For grouped clues we display 11.12,23 accross instead of just a number
-    &:before {
-        position: initial;
-    }
+//For grouped clues we display 11.12,23 accross instead of just a number
+.crossword__clue--display-group-order:before {
+    padding-right: .875em; // 14px - Matches the width between double digit numbers and clue of non-grouped clues
```

Comment:
Ah, I see! I didn't read that properly. Nice one.

On Tue, 15 Sep 2015 at 10:29 Gareth Trufitt <notifications@github.com>
wrote:

> In static/src/stylesheets/module/crosswords/_clues.scss
> <https://github.com/guardian/frontend/pull/10513#discussion_r39490638>:
>
> > @@ -36,12 +38,8 @@
> >      color: colour(neutral-2);
> >  }
> >
> > -.crossword__clue--display-group-order {
> > -
> > -    padding-left: 0px;
> > -
> > -    //For grouped clues we display 11.12,23 accross instead of just a number
> > -    &:before {
> > -        position: initial;
> > -    }
> > +//For grouped clues we display 11.12,23 accross instead of just a number
> > +.crossword__clue--display-group-order:before {
> > +    padding-right: .875em; // 14px - Matches the width between double digit numbers and clue of non-grouped clues
>
> Yup, 14px was this width
> [image: image]
> <https://cloud.githubusercontent.com/assets/638051/9872931/651f4cee-5b94-11e5-9b51-7ec0f857406e.png>
>
> Bit of a magic number, but stylistically matches better. I also used em
> because the padding and min-width values were also ems so it would scale
> proportionately to the li font-size, as opposed to the document - what do
> you reckon, leave as em or just use the explicit px width (which would
> convert to rem)?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/guardian/frontend/pull/10513/files#r39490638>.
>

-- 

------------------------------
Visit theguardian.com. On your mobile and tablet, download the Guardian 
iPhone and Android apps theguardian.com/guardianapp and our tablet editions 
theguardian.com/editions.  Save up to 57% by subscribing to the Guardian 
and Observer - choose the papers you want and get full digital access. 
 Visit subscribe.theguardian.com

This e-mail and all attachments are confidential and may also be 
privileged. If you are not the named recipient, please notify the sender 
and delete the e-mail and all attachments immediately. Do not disclose the 
contents to another person. You may not use the information for any 
purpose, or store, or copy, it in any way.  Guardian News & Media Limited 
is not liable for any computer viruses or other material transmitted with 
or as part of this e-mail. You should employ virus checking software.
 
Guardian News & Media Limited is a member of Guardian Media Group plc. Registered 
Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.  Registered 
in England Number 908396